azuredeploy.json

```diff
@@ -2,9 +2,45 @@
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "location": {
-            "type": "string"
-        },
+      "location": {
```

Comment:
I like the idea of setting this to a collection of locations but I'm going to be not all of these support container instances. Double check that these regions all can support ACI. 
```
az provider show --namesspace Microsoft.ContainerInstance -o table
```

Reply:
You were right. I took them from the error message I got from "someLocation". Updated to the result from that query.